bot-data.json

```diff
@@ -144,8 +146,11 @@
         },
         "TensorBot" : {"name": "TensorBot", "makers": ["Soren"], "language": "python", 
             "notes" : ["100% Machine Learning bot trained on Diablo recordings and built using TensorFlow"]
+        },
+        "BroccoliBot": {"name": "BroccoliBot", "makers": ["Ellbell"], "language": "python", 
+            "notes": ["Dis bot is p bad ngl. but it kinda knows how to position? so das good. EAT UR VEGGIES OR DIE!"]
         }
-    },
```

Comment:
Please don't remove this comma, the website would get upset. Looks good besides that!